scripts/ci/create-plunker-bundle.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * This script creates a bundle of all components and publishes it to Firebase.
+ * The bundle will be used to load a Plunker Demo of Angular Material 2.
+ */
+
+const spawn = require('child_process').spawnSync;

```

Comment:
Awesome! I'm available on Slack, if there're any questions (I got one :D).

> When deploying to firebase, we overwrite the current hosting and loose the old bundles (like for tags)
We need a way to pull the old files or find another approach for hosting